packages/client/src/Loc.ts

```diff
@@ -1138,6 +1145,27 @@ export class LegalOfficerPendingRequestCommands {
         }
         return await this.request.refresh() as AcceptedRequest | OpenLoc;
     }
+
+    async estimateFeesAccept(): Promise<FeesClass | undefined> {
+        this.request.ensureCurrent();
+        const requesterAccount = this.request.data().requesterAddress;
+        const requesterLoc = this.request.data().requesterLocId;
+        const sponsorshipId = this.request.data().sponsorshipId;
+        const locType = this.request.data().locType;
+        if(locType === "Transaction") {
+            return this.client.estimateFeesAcceptTransactionLoc({
+                locId: this.locId,
+                requesterAccount,
+                requesterLoc,
+            });
+        } else if(locType === "Identity") {
+            return this.client.estimateFeesAcceptIdentityLoc({
+                locId: this.locId,
+                requesterAccount,
+                sponsorshipId,
+            });
+        }
```

Comment:
Collection case is missing I think.

Reply:
Comment is added to explain why fees are always undefined when accepting a collection.